docs/source/models/concept_models/vivarium_alzheimers/concept_model.rst

```diff
@@ -281,23 +281,32 @@ The basic plan for the design of the simulation is as follows:
 5.1 Model Runs
 ------------------------
 
+.. _2025_alzheimers_model_runs_table:
+
 .. list-table:: Model run requests
   :header-rows: 1
 
   * - Number
     - Description
     - Scenarios
-    - Directory
     - Specification mods
     - Stratification mods
     - Observer mods
   * - 0.0
-    - Mock-up run with full population and fake data to test runtime
-    -
-    -
-    -
-    -
-    -
+    - Mock-up run with fake data but full population and mock-ups of all
+      components to test runtime
+    - ?
+    - Open cohort simulating entire population (including susceptible
+      simulants, not just simulants who will get AD) in all age groups;
+      simulants enter at age = 0 using crude birth rate
+    - Default
```

Comment:
@albrja Does this info for the specifications and stratifications look accurate for the mock-up run?